sdk/formrecognizer/ai-form-recognizer/review/ai-form-recognizer.api.md

```diff
@@ -0,0 +1,1299 @@
+                    };
+                };
+                readonly Departments: {
+                    readonly type: "array";
+                    readonly items: {
+                        readonly type: "string";
+                        readonly description: "Department or organization";
+                        readonly example: "Cloud & Al Department";
+                    };
+                };
+                readonly Addresses: {
+                    readonly type: "array";
+                    readonly items: {
+                        readonly type: "string";
+                        readonly description: "Address";
+                        readonly example: "4001 1st Ave NE Redmond, WA 98052";
+                    };
+                };
+                readonly WorkPhones: {
+                    readonly type: "array";
+                    readonly items: {
+                        readonly type: "phoneNumber";
+                        readonly description: "Work phone number";
+                        readonly example: "+1 (987) 213-5674";
+                    };
+                };
+                readonly MobilePhones: {
+                    readonly type: "array";
+                    readonly items: {
+                        readonly type: "phoneNumber";
+                        readonly description: "Mobile phone number";
+                        readonly example: "+1 (987) 123-4567";
+                    };
+                };
+                readonly Faxes: {
+                    readonly type: "array";
+                    readonly items: {
+                        readonly type: "phoneNumber";
+                        readonly description: "Fax number";
+                        readonly example: "+1 (987) 312-6745";
+                    };
+                };
+                readonly OtherPhones: {
+                    readonly type: "array";
+                    readonly items: {
+                        readonly type: "phoneNumber";
+                        readonly description: "Other phone number";
+                        readonly example: "+1 (987) 213-5673";
+                    };
+                };
+                readonly Emails: {
+                    readonly type: "array";
+                    readonly items: {
+                        readonly type: "string";
+                        readonly description: "Contact email";
+                        readonly example: "chris.smith@contoso.com";
+                    };
+                };
+                readonly Websites: {
+                    readonly type: "array";
+                    readonly items: {
+                        readonly type: "string";
+                        readonly description: "Website";
+                        readonly example: "https://www.contoso.com";
+                    };
+                };
+            };
+        };
+    };
+};
+
+// @public
+export interface CommonClientOptions extends PipelineOptions {
```

Comment:
`CommonClientOptions` is already defined in `core-client` so this definition could be confusing. Did you consider `FormRecognizerClientOptions`? similar to [TA's common op options one](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/textanalytics/ai-text-analytics/src/textAnalyticsOperationOptions.ts)

Reply:
FormRecognizerClientOptions is the name of the options type from the previous SDK, which I'd like to avoid using. Maybe `FormRecognizerCommonClientOptions` would be better, but is it really likely that this type would conflict/shadow the one from core-client? Is there any use case for a customer to import from core-client directly? They're not likely to import this type from us either.

Reply:
> is it really likely that this type would conflict/shadow the one from core-client? Is there any use case for a customer to import from core-client directly? They're not likely to import this type from us either.

No I was thinking from my POV. 

> FormRecognizerClientOptions is the name of the options type from the previous SDK, which I'd like to avoid using.

May I ask why?

> Maybe FormRecognizerCommonClientOptions would be better

Sounds good to me.

Reply:
> May I ask why?

We are explicitly supporting a side-by-side use case in which a customer may import and use both of the SDK versions if necessary, and I don't want someone to import/use `FormRecognizerClientOptions` from the new SDK thinking it means the same thing as it used to. I'm going to change this to `FormRecognizerCommonClientOptions`.